improver_tests/acceptance/test_generate_timezone_mask_ancillary.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2020 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Tests for the generate-timezone-mask-ancillary CLI."""
+
+import pytest
+
+from . import acceptance as acc
+
+pytest.importorskip("timezonefinder")
+pytest.importorskip("numba")
+pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
+
+CLI = acc.cli_name_with_dashes(__file__)
+run_cli = acc.run_cli(CLI)
+
+GRIDS = ["uk", "global"]
+TIMES = ["20200615T1200Z", "20201215T1200Z"]
+
+
+@pytest.mark.parametrize("time", TIMES)
+@pytest.mark.parametrize("grid", GRIDS)
+def test_ignoring_dst(tmp_path, time, grid):
+    """Test masks generated ignoring daylights savings time. The time of year
+    should have no impact on the result."""
+
+    kgo_dir = acc.kgo_root() / f"generate-timezone-mask-ancillary/{grid}/"
+    kgo_path = kgo_dir / "ignore_dst_kgo.nc"
+    input = kgo_dir / "input.nc"
```

Comment:
Codacy found an issue: [Redefining built-in 'input'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5933437)